workspaces/mta/.changeset/two-fishes-cough.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage-community/backstage-plugin-catalog-backend-module-mta-entity-provider': minor
+'@backstage-community/backstage-plugin-mta-backend': minor
+---
+
+Add configuration for github auth
```

Comment:
So this doesn't get shipped with the package so not sure this makes sense as what should show up in the changelog. Maybe it would be better to say "Removed verbose logging statements" or something along those lines?